slave/util.go

```diff
@@ -0,0 +1,19 @@
+package slave
+
+import (
+	"fmt"
+)
+
+// StrSlice a slide of string
+type StrSlice []string
+
+// StringInSlice checks if a string exists in a StrSlice.
+func StringInSlice(list StrSlice, a string) bool {
```

Comment:
`StringInSlice` is a very generic function name. The way this is using it is actually for validating that the iteration over assemblers is valid and the current assembler is one passed by the user. What do you think about implementing something like:
```go
func isValid(assembler string) bool {
    switch assembler {
        case MegaHit, 
                 Abyss, 
                 Flye:
            return true
        default:
            return false
    }
}
```
This can go in the `util.go` of the constants package so it has access to the assembler names.

Reply:
The validity is implicitly being checked in `slave.go` when we loop through the `constants.AvailableAssemblers`. The `StringInSlice` function merely ensure to use `k`-th assembler if and only if it is one of the assemblers that is given by users through YAML config file. The func name is too generic, I agree. How about the changes proposed in 329bfd3?

Reply:
Now that I think more about it, generic is fine. It's a simple check that does not need to be over-engineered.